### PR TITLE
3 Save Job Button Functionality

### PR DESCRIPTION
**Pull request by ZachariaLentz**
-Changed insert for Job_Posts table to reflect API data structure
-Edited init.sql to reflect API data structure for Job_Posts

### CODE REVIEW

**Review by maki-q — COMMENTED** [review]
Looks like good changes, but I'll wait for @Gobleizer's final review to sign off.

**Review by Gobleizer — APPROVED** [review]
Thank you for making these changes

**Pull request merged**
